src/MapfileStyleParser.ts

```diff
@@ -810,6 +810,13 @@ export class MapfileStyleParser implements StyleParser {
       mapfileLayerLabelItem = mapfileClass.text ? mapfileClass.text : mapfileLayerLabelItem;
       mapfileClass.labels.forEach((mapfileLabel) => {
         mapfileLabel.text = mapfileLabel.text ? mapfileLabel.text : mapfileLayerLabelItem;
+
+        // Set Icons in front of the associated label
+        const labelIcons = this.getIconsFromMapfileLabel(mapfileLabel);
+        labelIcons?.forEach(labelIcon => {
```

Comment:
Minor: `labelIcons?.forEach` -> `labelIcons.forEach`. `getIconsFromMapfileLabel` always returns an array.

Reply:
Thx, removed the optional mark.